build.gradle.kts

```diff
@@ -23,7 +23,6 @@ buildscript {
         classpath(libs.androidToolsGradlePlugin)
         classpath(libs.kotlinGradlePlugin)
         classpath(libs.kotlinSPGradlePlugin)
-        classpath(libs.ktLintGradlePlugin)
```

Comment:
I'm wondering what the flow will be. Before I was running `ktlintFormat` before pushing to git - at which step formatting will be done now? Also sometimes `ktlintFormat` fails to do the job, like in case of too long lines - in that case some manual changes were needed.

Reply:
You ca still run `ktlint -F "**/*.kt" "**/*.kts" '!**/build/generated/**' '!**/build/kspCaches/**'` or make it a pre-commit hook. Will behave the same.
However I guess I should update the doc with the current required ktlint version (0.45.1)

Reply:
but in this case I need to install ktlint CLI on my machine manually, do I get it right? Or you are using the Dockerimage for that which has a project folder mounted to it?

Reply:
I personally use the local ktlint CLI on my machine (the gradle plugin was just already a wrapper around the CLI on your machine no?)

Reply:
No, Gradle plugin is not a wrapper for the existing `ktlint`, I don't have `ktlint` installed on my machine. But I see the proposed approach now :)